### PR TITLE
Add Pendo analytics tool

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
https://github.com/gravitee-io/issues/issues/7781

**Description**

Add Pendo analytics tool

**Additional context**

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/7781-pendo-analytics/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-fkvqkzvmot.chromatic.com)

### CODE REVIEW

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml [hunk 1]
```diff
@@ -18,6 +18,15 @@
 #  api:
 #    url: http://localhost:8083/management
 
+# Active Pendo Analysis Tool
```
Comment (ThibaudAV) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml]: cci @mouligno

File: gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/settings/ConsoleAnalyticsPendo.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.gravitee.rest.api.model.annotations.ParameterKey;
+import io.gravitee.rest.api.model.parameters.Key;
+
+/**
+ * @author GraviteeSource Team
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ConsoleAnalyticsPendo {
```
Comment (gaetanmaisse) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/settings/ConsoleAnalyticsPendo.java:28]: As we are now using Lombok you can use the `@Data` annotation and so remove the getter/setter

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml [hunk 3]
```diff
@@ -18,6 +18,15 @@
...
+# To make our SaaS offering easier to use, we now have the ability to track activity in Pendo. This is disabled by
+# default, but if you'd like to help us in this journey, don't hesitate to reach us to get our key ;)
+#console:
```
Comment (gaetanmaisse) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml]: There is already a `console` key on line 15, so we can merge the analytics part in it?

Comment (ThibaudAV) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml]: there is another one at 373. that's why I added another one. but if it react you too I will merge the 3

File: gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/settings/ConsoleAnalyticsPendo.java (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/settings/ConsoleAnalyticsPendo.java:28]: nice !

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml (hunk 1 above)
Comment (mouligno) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml]: LGTM

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml [hunk 4]
```diff
@@ -11,13 +11,26 @@
...
-# Specify the URL of the Management API and UI of this instance, mandatory if you want to connect it to Cockpit
+# Console settings
 #console:
+## Specify the URL of the Management API and UI of this instance, mandatory if you want to connect it to Cockpit
 #  ui:
 #    url: http://localhost:3000
 #  api:
 #    url: http://localhost:8083/management
 
+## Active Pendo Analysis Tool
```
Comment (gaetanmaisse) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml]:
```suggestion
## Pendo Analysis Tool
```

**Review by gaetanmaisse — APPROVED** [review]

**Review by phiz71 — APPROVED** [review]

**Pull request merged**